cfme/tests/intelligence/reports/test_canned_corresponds.py

```diff
@@ -93,3 +94,97 @@ def test_cluster_relationships(soft_assert, setup_a_provider):
                     break
         else:
             soft_assert(False, "Hostname {} not found in {}".format(host_name, provider_name))
+
+
+@pytest.mark.tier(3)
+@test_requirements.report
+def test_operations_vm_on(soft_assert, setup_a_provider):
+
+    appliance = get_or_create_current_appliance()
```

Comment:
All provider objects should have an appliance attribute set, you should be able to get this from `provider.appliance.db`